addons/point_of_sale/static/src/js/models.js

```diff
@@ -601,7 +625,7 @@ exports.PosModel = Backbone.Model.extend({
                     reject();
                 };
                 self.company_logo.crossOrigin = "anonymous";
-                self.company_logo.src = '/web/binary/company_logo' + '?dbname=' + self.session.db + '&company=' + self.company.id + '&_' + Math.random();
+                self.company_logo.src = '/web/binary/company_logo' + '?dbname=' + self.session.db + '&company=' + self.company.id;
```

Comment:
probably the random value had been set to reload the picture of the logo if it had been changed. So we probably should set someting like the write_date of the company if fit is available.

Reply:
I replaced the random param with `'&write_date=' + self.company.write_date;`